src/components/SignIn/SignIn.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import SignIn from './SignIn';
+
+export default {
+  title: 'Platform/Sign In',
+  component: SignIn,
+};
+
+const Template = (args) => <SignIn {...args} />;
+
+export const WebexAuth = Template.bind({});
+WebexAuth.args = {
+  authUrl: 'https://webexapis.com/v1/authorize',
+  responseType: 'code',
+  clientId: 'C185d483590b3b2b3abe9537bd0307bec772b187aa544b4471dfe9339c983d249',
```

Comment:
do we not have env files for this information? Not sure if we should be hardcoding client ids here

Reply:
Correct, @kishorepmr why don't we get this value from env file?